Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanelModifier.swift

```diff
@@ -105,9 +105,6 @@ private struct FloatingPanelModifier<PanelContent>: ViewModifier where PanelCont
                     isPresented: isPresented,
                     content: panelContent
                 )
-                // When the panel is anchored to the bottom of the screen (compact) ignore the
-                // device's bottom safe area.
-                .ignoresSafeArea(.container, edges: isCompact ? .bottom : [])
```

Comment:
This has moved into the FloatingPanel view itself. It allows us to locate the two `ignoresSafeArea` modifiers right next to each other and factor in the state of the keyboard.